### ISSUE TITLE
Build warning due to critical dependency in ms-rest

### ISSUE THREAD

**Issue by mattias-kindborg-at-work** [Azure/azure-sdk-for-node#5202]
I'm building a React application using `ms-rest` for authentication to Azure, but get the following warning when building the application using [create-react-app](https://github.com/facebook/create-react-app).

```
./node_modules/ms-rest/lib/serviceClient.js
Critical dependency: the request of a dependency is an expression
```

I guess my first question is whether `ms-rest` is supported to run in browsers? If not, do you recommend me to use any other package to get the functionality of `TokenCredentials`?

**Comment by mattias-kindborg-at-work** [comment]
Hi @ramya-rao-a

The issue I was facing was due to my own fault. I was using package `azure-arm-resource` to list subscriptions, and used the class `TokenCredentials` from package `ms-rest` to create an instance of class `SubscriptionClient`.

However, it seems that package `azure-arm-resource` is deprecated in favor of packages scoped to `@azure`, thus I replaced `azure-arm-resource` with `@azure/arm-subscriptions` and used  `TokenCredentials` from `@azure/ms-rest-js`.

I'll close this issue.

Thanks for the quick respons, and have a really good day!
